shell/browser/api/atom_api_web_contents.cc

```diff
@@ -1286,31 +1286,26 @@ void WebContents::SetBackgroundThrottling(bool allowed) {
   background_throttling_ = allowed;
 
   auto* contents = web_contents();
```

Comment:
this is also not necessary, we don't check the return value of `web_contents()` anywhere else